OrbitCore/EventBuffer.h

```diff
@@ -15,43 +15,52 @@
 
 class EventBuffer {
  public:
-  EventBuffer() : m_MaxTime(0), m_MinTime(LLONG_MAX) {}
+  EventBuffer() : min_time_(0), max_time_(LLONG_MAX) {}
```

Comment:
Why was min and max reversed here?

Reply:
min_time_ should be initialized to LLONG_MAX so that the first time that we add will become the new minimum. Inversely for max_time_.

Reply:
Ouh, thanks for spotting this. Seems that I mixed them up while refactoring. Will change that back again.

Reply:
Done. thanks!